src/state/slices/swapSlice/selectors.ts

```diff
@@ -0,0 +1,26 @@
+import type { Swap } from '@shapeshiftoss/swapper'
+import { createSelector } from 'reselect'
+
+import { swapSlice } from './swapSlice'
+
+import { selectQuoteIdParamFromFilter } from '@/state/selectors'
+
+export const selectSwaps = swapSlice.selectors.selectSwaps
```

Comment:
No need for this, the whole reason why we have selectors in slices is so that we don't have to declare new ones

Reply:
Removed in [fbcd366](https://github.com/shapeshift/web/pull/9625/commits/fbcd366ded019ee7c8586022801e2a9625cc80dd)